.github/workflows/ci.yml

```diff
@@ -37,31 +37,32 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.6', '3.9']
+        python-version: ['3.7', '3.9']
         use-conda: ['Yes', 'No']
         include:
         - os: ubuntu-latest
           special-invocation: 'xvfb-run --auto-servernum '
-        - python-version: '3.6'
+        - python-version: '3.7'
           use-conda: 'No'
-          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.6 defaults channel
+          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.7 defaults channel
           pyside2-version: 5.12.0  # 5.12.1-5.12.6 fails on collection/segfaults on patch test
         - os: ubuntu-latest
-          python-version: '3.6'
+          python-version: '3.7'
           use-conda: 'No'
           skip-pyqt6: true  # No wheels on Py 3.6 Linux CIs
         - os: windows-latest
           python-version: '3.9'
           use-conda: 'No'
           pyside2-version: 5.15  # No 5.12 wheel on Windows and Python 3.9
         - os: windows-latest
-          python-version: '3.6'
-          use-conda: 'Yes'
-          pyqt5-qt-version: '5.9'  # 5.12 is apparently unreliable here
+          python-version: '3.7'
+          use-conda: 'No'
+          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.7 defaults channel
         - os: macos-latest
-          python-version: '3.6'
+          python-version: '3.7'
           use-conda: 'No'
-          skip-pyqt6: true  # No wheels on Py 3.6 macOS CIs
+          skip-pyqt6: true  # No wheels on Py 3.7 macOS CIs
+          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.7 defaults channel
```

Comment:
Again, this was already specified above (assuming you verified its necessary at all), so not sure why it is repeated here?